unit2-java/lesson16.rst

```diff
@@ -5,24 +5,19 @@ Goals
 -----
 
 - Understand how object-relational mapping allows object-oriented web applications to store data in relational databases
-- Implement ORM with one-to-one and one-to-many relationships in Spring Boot
+- Implement ORM for individual, non-related entities in Spring Boot
 
 Objectives
 ----------
 
 - Explain what object-relational mapping refers to
```

Comment:
I'm not sure of the grammar acceptability of ending a sentence with "to".  Perhaps, "...mapping means"?